griptape/drivers/image_query/anthropic_image_query_driver.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional
+from attr import define, field, Factory
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseImageQueryDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from anthropic import Anthropic
+
+
+@define
+class AnthropicImageQueryDriver(BaseImageQueryDriver):
+    """
+    Attributes:
+        api_key: Anthropic API key.
+        model: Anthropic model name.
+        client: Custom `Anthropic` client.
+        max_output_tokens: Max output tokens to return.
+    """
+
+    api_key: str = field(kw_only=True, metadata={"serializable": True})
```

Comment:
This should have a default set to `None`, so that it can be automatically loaded from an environment variable.

Reply:
I was following a standard that was set in the [AnthropicPromptDriver](https://github.com/griptape-ai/griptape/blob/dev/griptape/drivers/prompt/anthropic_prompt_driver.py#L24). I can update there as well to stay consistent? but I definitely think they need to follow the same pattern.

Reply:
I see Collin updated that in his [structure PR](https://github.com/griptape-ai/griptape/pull/702/files#diff-388135d66c742b44c2b3cd526877c0d0d34ca1938daa41523ade65a86a3f24f5R21) will follow that standard.